docs/notebooks/Common_Gotchas_in_JAX.md

```diff
@@ -421,6 +421,32 @@ outputId: e31b43b3-05f7-4300-fdd2-40e3896f6f8f
 jnp.sum(jnp.array(x))
 ```
 
+Of course, writing the conversion explicitly does not make it any faster. Consider the following example of converting a nested list of integers to a JAX array:
```

Comment:
I think this could be made much more concise, to be honest. You could say something along the lines of

> keep in mind that although `jnp.array()` accepts lists of values for convenience, passing large lists to `jnp.array()` can incur the same performance penalty that JAX prevents by disallowing list inputs to `jnp.sum` and other functions. If you are converting a large list of scalars to a JAX array outside the context of `jit` or other transforms, it can be far more performant to instead use `numpy` to convert the list to an array; e.g. `np.array(x)` rather than `jnp.array(x)`."

Less is more for these kinds of docs, I think.

Reply:
Sure, I can condense the writing. You're right that this page is already quite long and it shouldn't be filled with too much detail.
What do you think about the timeit-example? I think it's instructive to directly see just how much faster numpy is than JAX in this context.
And the last part (about the casting) isn't really a part of the performance issue, though it is related and maybe highlights a different "sharp edge". Should I cut this part?